tests/common.py

```diff
@@ -418,7 +418,7 @@ def setUpClass(cls):
   def setUp(self):
     super().setUp()
     self.settings_mods = {}
-    self.emcc_args = ['-Werror']
+    self.emcc_args = ['-Werror', '-Wno-limited-postlink-optimizations']
```

Comment:
Can we also have a test showing that the new warning is produced correctly?

Reply:
Good idea.